### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by abdulsattar**
## Details
Update dependencies.

## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,28 +38,28 @@
...
         "bytes": "^3.1.2",
         "es5-proxy-compat": "^0.22.4",
-        "eslint": "^8.35.0",
+        "eslint": "^8.36.0",
         "eslint-plugin-import": "^2.27.5",
         "eslint-plugin-jest": "^27.1.7",
-        "fs-extra": "^11.1.0",
-        "glob": "^8.1.0",
+        "fs-extra": "^11.1.1",
+        "glob": "^9.3.1",
```
Comment (nolanlawson) [inline package.json:51]: Based on the [release notes for glob 9.0.0](https://github.com/isaacs/node-glob/blob/main/changelog.md#90), as well as the fact that we are using `glob.sync` in [several of our scripts](https://github.com/search?q=repo%3Asalesforce%2Flwc%20glob.sync&type=code) (e.g. releasing a new version, running benchmarks), and given that [`glob.sync` does not seem to be exported anymore](https://unpkg.com/browse/glob@9.3.1/dist/cjs/glob.d.ts), this 9.0 upgrade seems likely to break in a way that would not be detected by our CI tests.

File: scripts/release/version.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ function getPackagesToUpdate() {
 
     const workspacePkgs = rootPackageJson.workspaces.reduce(
         (accWorkspacePkgs, workspace) => {
-            const workspacePkg = glob.sync(`${workspace}/package.json`);
+            const workspacePkg = globSync(`${workspace}/package.json`);
```
Comment (nolanlawson) [inline scripts/release/version.js:83]: We are using `glob.sync` in [a few other places as well](https://github.com/search?q=repo%3Asalesforce%2Flwc%20glob.sync&type=code).

**Review by nolanlawson — APPROVED** [review]

File: packages/@lwc/template-compiler/src/parser/constants.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ const ATTRIBUTE_NAME_CHAR = [
     '\\-.0-9\\u00B7\\u0300-\\u036F\\u203F-\\u2040',
 ].join('');
 
+// eslint-disable-next-line no-misleading-character-class
```
[the remark this answers is not in the document]
Comment (abdulsattar) [reply packages/@lwc/template-compiler/src/parser/constants.ts:32]: I have this stackblitz for all the codepoints that combine into a single grapheme:
https://stackblitz.com/edit/node-pepzk5?file=index.js

I decided to fix this later, so, just disabled the warning for now.

[the remark this answers is not in the document]
Comment (abdulsattar) [reply packages/@lwc/template-compiler/src/parser/constants.ts:32]: I have this stackblitz for all the codepoints that combine into a single grapheme:
https://stackblitz.com/edit/node-pepzk5?file=index.js

I decided to fix this later, so, just disabled the warning for now.

**Pull request merged**